demos/demos_databases_apis/spanner/google_spanner_finance_graph.ipynb

```diff
@@ -0,0 +1,213 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "9418e0e1-648d-4cd7-a264-9bf910cbcfe3",
+   "metadata": {},
+   "outputs": [],
+   "source": [
```

Comment:
first cell should be a description

later cell should be install command (`pip install graphistry[spanner_gsql]` ?)